src/package.json

```diff
@@ -143,7 +143,8 @@
     "vue-gtag": "2.0.1",
     "vue-qrcode-reader": "5.3.4",
     "vue-tsc": "1.8.10",
-    "workbox-precaching": "7.0.0"
+    "workbox-precaching": "7.0.0",
+    "@tiptap/extension-text-align": "^2.1.8"
```

Comment:
`pnpm add -DE ...`: `-D` installs it as `devDependency` `-E` as exact version (without the `^`, is taken care of by renovate).
As this package does not need to provide its dependencies to other packages, `devDependency`s are fine.